Pilot_Plant_Dataset_Doc.ipynb

```diff
@@ -0,0 +1,2231 @@
+{
```

Comment:
<h3>1b. Glycol Pilot Plant Modes, Run <em>Statuses,</em> and Alarms  </h3><p> </p><p> <strong>[TO REVIEW]</strong> &nbsp;</p><p> </p><p>To identify what was going at a given time at the pilot plant, a few select <strong><em>digital state data streams</em></strong> are important. Unlike data streams with float values, the values of <strong><em>digital state data</em></strong> streams comes from a discrete set of strings, each string having<strong><em> a unique integer code</em></strong> within the set. <strong><em>Each</em></strong> digital state stream is associated <strong><em>with</em></strong> a digital state <strong>set,</strong> and <strong><em>one</em></strong> digital state set may be used by multiple <strong><em>data</em></strong> streams. <strong><em>The most basic example of a digital state set is Off/On, where 0 = Off and 1 = On. </em></strong></p><p> </p><p><strong><em>Listed here are</em></strong> the digital state <strong><em>sets</em></strong> shared by all mode streams, run status streams, and alarms streams respectively: </p><p> </p><p><strong>Digital State Set: delmar.ControllerMode</strong> </p><p>| Digital State Name | Code | </p><p>|:------------------:|:----:| </p><p>| Init. Manual       | 2 |  </p><p>| Manual             | 8 |  </p><p>| Auto | 16 |  </p><p>| Cascade | 32 |  </p><p><strong>Digital State Set: delmar.RunStatus</strong> </p><p>| Digital State Name | Code | </p><p>|:------------------:|:----:| </p><p>| Not Running       | 0 |  </p><p>| Running           | 1 |  </p><p><strong>Digital State Set: delmar.AlarmStatus</strong> </p><p>| Digital State Name | Code | </p><p>|:------------------:|:----:| </p><p>| Normal | 0 | </p><p>| Alarm  | 1 |  </p><p> </p><p>In <a href="https://app.reviewnb.com/academic-hub/datasets/pull/1/files/#section_5" target="_blank">Section 5</a>, all column names having "Mode" in them and Category as the Stream_Type have values from the <strong>delmar.ControllerMode</strong> table. All column names having "Run Status" as a substring and Category as the Stream_Type have value from the <strong>delmar.RunStatus</strong>, and similarly "Alarm" in the column name indicates values from the <strong>delmar.AlarmStatus</strong> set. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/academic-hub/datasets/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
done